tests/ucx_multi_node.sh

```diff
@@ -0,0 +1,71 @@
+export BINARY=${BINARY:-./test_benchmark_stress}
+export ARGS=${ARGS:-30000000 1000000000 0}
+
+function cleanup() {
+    echo "kill all testing process of ps lite for user $USER"
+    if [[ $EUID -ne 0 ]]; then
+        pkill -9 -u $USER -f $BINARY
+    else
+        pkill -9 -f $BINARY
+    fi
+    sleep 1
+}
+trap cleanup EXIT
+# cleanup # cleanup on startup
+
+export DMLC_NUM_WORKER=${DMLC_NUM_WORKER:-1}
+export DMLC_NUM_SERVER=$DMLC_NUM_WORKER
+export CUDA_HOME=
+export UCX_HOME=
+export LD_LIBRARY_PATH=$CUDA_HOME/lib64:$UCX_HOME/lib
+
+export DMLC_PS_ROOT_URI=${NODE_ONE_IP}
+export BYTEPS_ORDERED_HOSTS=${NODE_ONE_IP},${NODE_TWO_IP}
+
+export DMLC_PS_ROOT_PORT=${DMLC_PS_ROOT_PORT:-12279} # scheduler's port (can random choose)
+export UCX_IB_TRAFFIC_CLASS=236
+export DMLC_INTERFACE=eth2        # my RDMA interface
+export UCX_TLS=ib,cuda
```

Comment:
better not specify it, UCX has to select transports out of the box

Reply:
Sure, I'll remove it

Reply:
removed